### PR TITLE
Fix calculator-online.net

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/AdguardTeam/AdguardFilters/issues/95200

### CODE REVIEW

File: EnglishFilter/sections/content_blocker.txt [hunk 1]
```diff
@@ -85,6 +85,7 @@ cdn.braun634.com^$third-party
 @@||zoover.adtrackers.net/shared/bannerpages/darttagsbanner.aspx
 @@||zoover.bannertrack.net/shared/bannerpages/darttagsbanner.aspx
 !
+@@||calculator-online.net^$generichide
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/content_blocker.txt:88]: this rule doesn't work

File: EnglishFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -623,6 +623,8 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/95200
+calculator-online.net#%#//scriptlet('prevent-setTimeout', 'adsbygoogle')
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/antiadblock.txt:627]: better:
```
||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,domain=calculator-online.net,important
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=calculator-online.net
```

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/antiadblock.txt (hunk 2 above)
Comment (piquark6046) [reply EnglishFilter/sections/antiadblock.txt:627]: On Firefox with AdGuard Browser Extension, These don't work well.

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/antiadblock.txt:627]: @piquark6046 
I don't see the adsbygoogle request at all in Firefox and I don't see antiadblock either. What do you see in Firefox?

**Pull request merged**